src/esp/metadata/attributes/AttributesEnumMaps.h

```diff
@@ -5,6 +5,7 @@
 #ifndef ESP_METADATA_ATTRIBUTES_ATTRIBUTESENUMMAPS_H_
 #define ESP_METADATA_ATTRIBUTES_ATTRIBUTESENUMMAPS_H_
 
+#include <Magnum/Trade/LightData.h>
```

Comment:
Crap, I should extract the enum outside, to a `Trade::LightType` so it can be forward-declared. This isn't good.

Reply:
I was going to refactor all the lights to use Magnum lights, but I don't know if I will get to it in here.

Reply:
Right, then just put that aside until I fix this ;)

Reply:
Ok, I've reverted all the light stuff